src/blog/2024/04/flowfuse-dedicated.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Introducing FlowFuse Dedicated
+subtitle: The Next Level in Node-RED Solutions
+description: FlowFuse Dedicated a new Node-RED solution with a secure, compliant, and fully managed single-tenant SaaS platform tailored for enterprise needs.
+date: 2024-04-22
+authors: ["marian-demme"]
+#image: /blog/2024/04/images/node-red-architecture.png
+tags:
+  - posts
+  - node-red
+  - flowfuse
+---
+
+Today, we are excited to officially announce FlowFuse Dedicated, a new way to use our enterprise platform as a single-tenant SaaS offering. This new offering provides all of the benefits of FlowFuse enterprise, with an added focus on data residency, isolation, and private networking to meet compliance needs.
+
+<!--more-->
+
+At FlowFuse, we cater to a diverse spectrum of clients, ranging from burgeoning startups to formidable global enterprises. It's clear that a one-size-fits-all approach to deployment doesn't align with the varied needs of our customer base. The stakes for compliance and security have never been higher. Even sectors not traditionally bound by stringent regulatory frameworks are finding compliance to be a critical factor impacting productivity and profitability.
+
+Here’s what sets FlowFuse Dedicated apart:
+
+- **Single-Tenant FlowFuse Platform:** Enjoy the exclusivity of a dedicated environment, ensuring that your operations are isolated, secure, and tailored to your specific needs.
+- **SaaS-like Efficiency:** Experience the ease and simplicity of a SaaS solution, where FlowFuse takes care of the entire management and deployment process. When software is not upgraded to the latest versions, organizations use obsolete and inefficient software that can be exposed to security threats. Because FlowFuse Dedicated is fully managed by FlowFuse, customers get access to the latest software features and security updates imminently.
+- **Regional Deployment Flexibility:** FlowFuse Dedicated can be deployed in the region of your choice, ensuring data residency requirements are met while optimizing performance by reducing latency. FlowFuse Dedicated can be deployed in 30+ regions.
+- **Enterprise-Grade Features:** Enjoy access to all enterprise features and configurations as your own FlowFuse Administrator, enabling you to manage multiple teams effectively.
+- **Comprehensive Compliance:** Meet complex compliance standards with ease, thanks to full data and source code IP isolation.
+
+FlowFuse Dedicated represents a paradigm shift in Node-RED offerings, a solution that not only meets the demanding requirements of today's enterprises in terms of speed, efficiency, and productivity but also navigates the complex terrain of compliance and data security with unparalleled finesse. If you are intrested in FlowFuse Dedicated, [contact us](https://flowfuse.com/contact-us/).
```

Comment:
link to new landing page as the CTA to contact

Reply:
Here's the PR for that page:https://github.com/FlowFuse/website/pull/1954

Reply:
```suggestion
FlowFuse Dedicated represents a paradigm shift in Node-RED offerings, a solution that not only meets the demanding requirements of today's enterprises in terms of speed, efficiency, and productivity but also navigates the complex terrain of compliance and data security with unparalleled finesse. [Learn more about FlowFuse Dedicated](https://flowfuse.com/dedicated/).
```